### ISSUE TITLE
improve accessibility of `<figure>` and `<caption>`

### ISSUE THREAD

**Issue by virginiaMae** [Doenet/DoenetML#651]
When a screen reader reads off a caption for a figure, it reads it off after the alt text. This reading order should be reversed, first the word "Figure" should be read off, then the caption, followed by "image" or "graph" (depending on the figure's children) then at last the alt-text provided by the description attribute. 
To accomplish this, when the html is generated, it should place the html component `<figcaption>` as the first child of the `<figure>` component. Currently, it is placed last. 
Here is a short example:
`<title>Images and Figures</title>

<figure>
  <caption>A squirrel making a valiant effort</caption>
  <image source="https://upload.wikimedia.org/wikipedia/commons/5/5a/Sciurus_carolinensis_-British_Columbia%2C_Canada-8.jpg" description="a brown squirrel on a tree hangs upside down and reaches precariously towards a hanging birdfeeder"/>
</figure>`
